docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3.9'
+
+services:
+  server:
+    image: grasscutter:1.0
+    build: .
+    volumes:
+      - ./resources:/app/resources
```

Comment:
considering exposing `config.json`

Reply:
> considering exposing `config.json`

Did you mean `<repo root>/config.json`?
Is this good to you? exposing everything might be configured
```
volumes:
  - ./resources:/app/resources
  - ./data:/app/data
  - ./config.json:/app/config.json
  - ./keystore.p12:/app/keystore.p12
```

but there's one more thing, docker will create a folder named `config.json` silently if `config.json` file not exist
which is really confused for someone don't create config file first
since `config.json` is in `.gitignore`

Reply:
And.. maybe a pre-created empty `resources` folder?

Reply:
resources folder gets created when grasscutter.jar gets run first time

Reply:
> resources folder gets created when grasscutter.jar gets run first time

`resources` folder creation is not a big deal, docker will create it automatically.
`config.json` not existed at the beginning is the real problem, which will cause docker create `config.json` **folder**